java/bulk-export/bulk-export-core/src/main/java/sleeper/bulkexport/model/BulkExportLeafPartitionQuerySerDe.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2022-2024 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sleeper.bulkexport.model;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+/**
+ * Serialises an BulkExportLeafPartitionQuery to and from JSON.
+ */
+public class BulkExportLeafPartitionQuerySerDe {
+    private final Gson gson;
+    private final Gson gsonPrettyPrinting;
+
+    public BulkExportLeafPartitionQuerySerDe() {
+        GsonBuilder builder = new GsonBuilder()
+                .serializeNulls();
```

Comment:
It looks like it's unnecessary to serialize nulls, since all the fields are mandatory.

Reply:
Removed